generator/beat/{beat}/magefile.go

```diff
@@ -0,0 +1,73 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
```

Comment:
Should we remove headers here too?

Reply:
Maybe? That file isn't really meant to be touched by users, and it's all "our" code.

Reply:
It'd be consistent with other go files around here, and I think that community beats shoudn't have files with header licenses `Licensed to Elasticsearch B.V.` in any case.

Reply:
Hm, good point.